samples/bot-types-of-cards/nodejs/Readme.md

```diff
@@ -0,0 +1,136 @@
+---
+page_type: sample
+description: Demonstrating the feature where user can send types of cards using bot.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+ contentType: samples
+ createdDate: "01/13/2023 05:00:17 PM"
+urlFragment: officedev-microsoft-teams-samples-bot-types-of-cards-nodejs
+
+---
+# Types of Cards
+
+This sample shows the feature where user can send types of cards using bot.
+
+## Interaction with app
+
+![Types Of Cards](Images/TypesOfCards.gif)
+
+## Prerequisites
+
+-  Microsoft Teams is installed and you have an account (not a guest account).
+-  To test locally, [NodeJS](https://nodejs.org/en/download/) must be installed on your development machine (version 16.14.2  or higher).
+-  [ngrok](https://ngrok.com/) or equivalent tunneling solution.
+
+## Setup
+
+> Note these instructions are for running the sample on your local machine, the tunnelling solution is required because
+> the Teams service needs to call into the bot.
+
+1. Register a new application in the [Azure Active Directory – App Registrations](https://go.microsoft.com/fwlink/?linkid=2083908) portal.
+
+2. Setup for Bot
+- Register Azure AD application resource in Azure portal
```

Comment:
Why this step again, it's already mentioned in 1st step.

Reply:
fixed